frontend/src/App.jsx

```diff
@@ -1,22 +1,49 @@
 import MapComponent from './map';
 import EditMenu from './edit';
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 
-export default function App () {
+import { getAllScenarios } from './requests';
 
+export default function App() {
   const [editMenuIsOpen, setEditMenuIsOpen] = useState(false);
+  const [parcelCoords, setParcelCoords] = useState(null);
+  const [parcelID, setParcelID] = useState(null);
+  const [savedScenarios, setSavedScenarios] = useState(null);
 
   const toggleEditMenu = () => {
     setEditMenuIsOpen((prev) => !prev);
-  }
+  };
+
+  useEffect(async () => {
+    const scenarios = await getAllScenarios();
+    setSavedScenarios(scenarios);
+  }, []);
+
+  const refreshSavedScenarios = async () => {
+    const scenarios = await getAllScenarios();
+    setSavedScenarios(scenarios);
+  };
 
-  console.log('render', editMenuIsOpen);
   return (
     <div className="App">
       <div className="map-and-menu-container">
-        <MapComponent toggleEditMenu={toggleEditMenu} />
-        <EditMenu open={editMenuIsOpen} />
+        <MapComponent
+          toggleEditMenu={toggleEditMenu}
```

Comment:
Yeah I did this for convenience because I wanted the Edit menu always open while I was developing. And I think it might make sense to stick with that for the time being. If users see nothing but a map, they won't know what to do or where to start. 

Do you think we should remove this function and the popup button for now? We could consider removing the popup as well, just by moving it's content over to the sidebar. I kind of prefer maps where the "popup" info appears in it's own box on the side rather than on top of the map where it can obscure view.

Reply:
Yeah let's remove the unused function and button just so it doesn't confuse us later. I'm fine with removing the popup too.